deployments/bridges/rialto-millau/entrypoints/relay-messages-to-millau-generator-entrypoint.sh

```diff
@@ -75,6 +78,7 @@ do
 			--lane $MESSAGE_LANE \
 			--origin Target \
 			remark \
+			--remark-payload $PAYLOAD \
```

Comment:
There are a few issues with this (which are mostly introduced in other PRs, but probably must be fixed here):
1) `remark-payload` is marked as [`conflicts_with("remark_size")`](https://github.com/paritytech/parity-bridges-common/blob/4d30e66cd7c322a81206a0e2eac21e9d295cca8d/relays/bin-substrate/src/cli/encode_call.rs#L46), so you can't (actually can, but see (2)) use both here - in this case the `--remark_size` [will be ignored](https://github.com/paritytech/parity-bridges-common/blob/4d30e66cd7c322a81206a0e2eac21e9d295cca8d/relays/bin-substrate/src/cli/encode_call.rs#L168);
2) [`conflicts_with("remark_size")`](https://github.com/paritytech/parity-bridges-common/blob/4d30e66cd7c322a81206a0e2eac21e9d295cca8d/relays/bin-substrate/src/cli/encode_call.rs#L46) and [`conflicts_with("remark_payload")`](https://github.com/paritytech/parity-bridges-common/blob/4d30e66cd7c322a81206a0e2eac21e9d295cca8d/relays/bin-substrate/src/cli/encode_call.rs#L168) should actually be `conflicts_with("remark-size")` and `conflicts_with("remark-payload")` (i.e. underscore -> hyphen). Right now this declaration doesn't work;
3) since `remark_payload` is now required, I don't see any point in keeping [this payload generation](https://github.com/paritytech/parity-bridges-common/blob/4d30e66cd7c322a81206a0e2eac21e9d295cca8d/relays/bin-substrate/src/cli/encode_call.rs#L191). But actually I'd prefer to keep this and mark `remark_payload` as `Optional<>`.